stage4/index1.html

```diff
@@ -0,0 +1,116 @@
+      <input type="text" id="question" class="question" required />
+
+      <label for="option1">Option 1</label>
+      <input type="text" id="option1" class="option" required />
+
+      <label for="option2">Option 2</label>
+      <input type="text" id="option2" class="option" required />
+
+      <label for="option3">Option 3</label>
+      <input type="text" id="option3" class="option" required />
+
+      <label for="option4">Option 4</label>
+      <input type="text" id="option4" class="option" required />
+
+      <select id="correctOption" required>
+        <option value="1">Option 1</option>
+        <option value="2">Option 2</option>
+        <option value="3">Option 3</option>
+        <option value="4">Option 4</option>
+      </select>
+      <label for="correctOption">Correct Option</label>
+
+      <button type="button" onclick="randomizeOptions()">
+        Randomize Options
+      </button>
+      <button type="button" onclick="saveQuestion()">Save Question</button>
+
+      <div id="question-list"></div>
+
+      <input type="submit" value="Submit" style="font-size: 20px" />
+
+      <button type="button" onclick="addQuestion()">Question</button>
+      <button type="button" onclick="randomSort()">Random Sort</button>
+      <button type="button" onclick="alphabeticalSort()">
+        Alphabetical Sort
+      </button>
+    </form>
+
+    <div id="player-section">
+      <h2 style="color: white">Player Section</h2>
+      <input type="text" id="player1Name" placeholder="Player 1 Name" />
+      <input type="text" id="player2Name" placeholder="Player 2 Name" />
+      <button type="button" onclick="startQuiz()">Start Quiz</button>
+
+      <div id="players">
+        <div id="player1" style="color: white; display: none">
+          <span id="player1name" style="color: white">...</span>
+          <span id="score1" style="color: white">0</span>
+          <button onclick="updateScore(1, true)">Correct</button>
+          <button onclick="updateScore(1, false)">Wrong</button>
+          <input
+            type="number"
+            class="score-input"
+            id="scoreInput1"
+            value="0"
+            min="0"
+            max="10"
+          />
+        </div>
+
+        <div id="player2" style="color: white; display: none">
+          <span id="player2name" style="color: white"></span>
+          <span id="score2" style="color: white">0</span>
+          <button onclick="updateScore(2, true)">Correct</button>
+          <button onclick="updateScore(2, false)">Wrong</button>
+          <input
+            type="number"
+            class="score-input"
+            id="scoreInput2"
+            value="0"
+            min="0"
+            max="10"
+          />
+        </div>
+      </div>
+
+      <div id="quiz-end">Quiz Ended! <span id="winner"></span></div>
+
+      <audio
+        id="correctSound"
+        src="https://www.soundjay.com/button/sounds/button-10.mp3"
```

Comment:
Think there's some problems with using shoudjay. It seems it's not available publicly.. Try this page instead:

You could try something like this instead:

https://simpleguics2pygame.readthedocs.io/en/latest/_static/links/snd_links.html